### ISSUE TITLE
Flex-Wrapping: False Positive When Using Flex-Flow

### ISSUE THREAD

**Issue by yuschick** [yuschick/stylelint-plugin-defensive-css#13]
**Describe the bug**
When using the `flex-flow` property and defining it to `column`, the `flex-wrapping` warning is still shown. However, when a flex in in any form of a column, the warning shouldn't be triggered.

**To Reproduce**

```css
.content {
  display: flex;
  flex-flow: column;
}

.content-reverse {
  display: flex;
  flex-flow: column wrap;
}
```

**Expected behavior**
When using `flex-flow` there should only be a warning from Stylelint if the direction is set to `row` and no wrap shorthand value is provided.

```css
/* No linter warning */
.content {
  display: flex;
  flex-flow: column; 
}

/* No linter warning */
.content {
  display: flex;
  flex-flow: row wrap; 
  flex-flow: row nowrap; 
  flex-flow: row wrap-reverse; 
}

/* No linter warning */
.content {
  display: flex;
  flex-flow: row;
  flex-wrap: wrap, wrap-reverse, nowrap;
}

/* Yes linter warning */
.content {
  display: flex;
  flex-flow: row; 
}
```

**Comment by yuschick** [comment]
Resolved in 0.8.0
